arch-sign-modules/PKGBUILD

```diff
@@ -1,17 +1,17 @@
 # Maintainer: Stuart Cardall <developer__at__it-offshore.co.uk>
 pkgname=arch-sign-modules
 _pkgname=Arch-SKM
-pkgver=0.2.3
-pkgrel=1
-pkgdesc="Helper scripts for Signed Kernel Modules"
+pkgver=0.2.51
+pkgrel=0
+pkgdesc="Signed (In Tree & Out of Tree) Kernel Modules for linux-lts linux-hardened linux-zen"
```

Comment:
many thanks for the `PR` - can you amend please to:

`pkgdesc="Signed (In Tree & Out of Tree) Kernel Modules for linux linux-lts linux-hardened linux-zen"`

(`linux` can also be built)